spring-backend/src/main/java/com/perfect8/blog/service/PostService.java

```diff
@@ -20,7 +21,13 @@ public PostService(PostRepository postRepository) {
         this.postRepository = postRepository;
     }
 
-    /// Get a post by its unique URL slug.
+
+public List<PostEntity> getAllPosts() {
+    return (List<PostEntity>) postRepository.findAll();
+}
+
+
+/// Get a post by its unique URL slug.
```

Comment:
Det blev fel på indenteringen här tror jag.

```suggestion
    public List<PostEntity> getAllPosts() {
        return (List<PostEntity>) postRepository.findAll();
    }


    /// Get a post by its unique URL slug.
```